netbox/utilities/utils.py

```diff
@@ -159,6 +159,9 @@ def serialize_object(obj, resolve_tags=True, extra=None):
         for field in ['level', 'lft', 'rght', 'tree_id']:
             data.pop(field)
 
+    if get_config().CHANGELOG_SKIP_EMPTY_CHANGES and 'last_updated' in data:
+        data.pop('last_updated')
+
```

Comment:
IMO we should move this logic into the model itself. We might use `serialize_object()` for other purposes irrelevant to change logging.